code/modules/virus2/wordlists.dm

```diff
@@ -0,0 +1,134 @@
+//For lists of words to be used with viruses.
```

Comment:
Try putting this in a folder in virus2 - maybe an "effect" folder, and move effect.dm into it as well?